tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,py35,pypy,pypy3
```

Comment:
Woops, I didn't mean to include that.  End of day mistake.

> On Apr 19, 2016, at 6:32 PM, Clinton Blackburn <notifications@github.com> wrote:
> 
> In tox.ini:
> 
> > @@ -1,10 +1,12 @@
> >  [tox]
> > -envlist = py26,py27,py32,py33,py34,py35,pypy,pypy3
> Why the changes to this file?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>